Spec/RealtimeClientConnection.swift

```diff
@@ -1660,6 +1660,34 @@ class RealtimeClientConnection: QuickSpec {
 
                 }
 
+                // RTN15f
+                it("ACK and NACK responses for published messages can only ever be received on the transport connection on which those messages were sent") {
+                    let options = AblyTests.commonAppSetup()
+                    options.disconnectedRetryTimeout = 0.5
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.close() }
+                    let channel = client.channels.get("test")
+                    var resumed = false
+                    waitUntil(timeout: testTimeout) { done in
+                        client.connection.once(.Connected) { _ in
+                            channel.publish(nil, data: "message") { _ in
```

Comment:
I don't understand @mattheworiordan 's point either. This looks good to me. The message is sent in one transport, which is immediately dropped. The fact that publish's callback is called _after_ the connection is resumed proves that the library resent the message in the new transport.